ruby/travis-test.sh

```diff
@@ -18,7 +21,7 @@ test_version() {
        which ruby && \
        gem install bundler && bundle && \
        rake test &&
-       cd ../conformance && make test_ruby"
+       RUBYLIB=lib:../conformance ../conformance/conformance-test-runner --failure_list ../conformance/failure_list_ruby.txt ../conformance/conformance_ruby.rb"
```

Comment:
It does still work. I changed this because this PR is trying to make it so multiple Ruby tests can be run in parallel. We do this by copying all the ruby code to directories `ruby-1.9`, `ruby-2.0`, etc. If the conformance tests always point back to the `ruby` directory, they don't support these parallel runs.

Your question made me think of a different way of solving the problem though that keeps the use of `make test_ruby`. Take a look and see what you think.